include/openssl/tls1.h

```diff
@@ -355,6 +355,9 @@ __owur int SSL_check_chain(SSL *s, X509 *x, EVP_PKEY *pk, STACK_OF(X509) *chain)
 # define SSL_CTX_get_tlsext_status_type(ssl) \
         SSL_CTX_ctrl(ssl,SSL_CTRL_GET_TLSEXT_STATUS_REQ_TYPE,0,NULL)
 
+# define SSL_CTX_IS_DTLS(ctx) \
+    (((ctx)->method->ssl3_enc->enc_flags & SSL_ENC_FLAG_DTLS) != 0)
+
```

Comment:
This macro does not belong here because it is accessing internal members of private structs. I think it should go into ssl_local.h

Reply:
Good point!